### DISCUSSION TITLE
the App for Windows, does not download. the download link does not work.

### DISCUSSION THREAD

**Discussion by rezamoogooei** [Chia-Network/chia-blockchain#5019] [Support]
the App for Windows, does not download. the download link does not work.

**Comment by moshenskyDV** [comment]
@rezamoogooei what exactly link doesn't work? Could you provide screenshot or add more details (link or something)?

**Reply by rezamoogooei** [reply]
I use this link:
https://download.chia.net/latest/Setup-Win64.exe

this is screenshot:
![image](https://user-images.githubusercontent.com/84204090/118334866-aca66180-b523-11eb-90c2-59f9e69764c9.png)

**Reply by moshenskyDV** [reply]
I just clicked on your link and download started correctly. Plobably something on your side. Try from another browser...

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
